source/Radar.cpp

```diff
@@ -33,6 +33,7 @@ const int Radar::SPECIAL = 5;
 const int Radar::ANOMALOUS = 6;
 const int Radar::BLINK = 7;
 const int Radar::VIEWPORT = 8;
+const int Radar::STAR = 9;
```

Comment:
you also need to add this to radar.h, look in line 39

Reply:
Alright, got it.